src/main/java/wolox/training/models/constans/ErrorConstants.java

```diff
@@ -0,0 +1,17 @@
+package wolox.training.models.constans;
+
+/**
+ *  Class that Contains the messages of the pre-validators errors
+ */
+public class ErrorConstants {
+
+    /**
+     * Preconditions message
+     */
+    public static String NOT_NULL = "Please check the %s supplied, its null!.";
+    public static String NOT_EMPTY = "Please check the %s supplied, its empty!.";
+    public static String NOT_NUMERIC = "The %s isn't numeric.";
+    public static String NOT_GREATER_THAN = "The %s cannot smaller that %s.";
+    public static String NOT_LATER_CURRENT_DATE = "The %s cannot be later than the current date.";
```

Comment:
This field is not used

Reply:
@JaimLus  It will be used when main makes the user merge, in the next card I will do that implementation or create a PR only for user validations